src/Validus/Validus.fs

```diff
@@ -622,13 +631,13 @@ module Validators =
 /// Custom operators for ValidationResult
 module Operators =
     /// Alias for ValidationResult.apply
-    let inline (<*>) f x = ValidationResult.apply f x
+    let inline (<*>) applier result = ValidationResult.apply applier result
 
-    /// Alias for Result.map
-    let inline (<!>) f x = Result.map f x
+    /// Alias for Validator.chain forward
+    let inline (>=>) v1 v2 = Validator.chain v1 v2
 
-    /// Alias for ValidationResult.bind
-    let inline (>>=) x f = Result.bind f x
+    /// Alias for Validator.chain backward
+    let inline (<=<) v1 v2 = Validator.chain v2 v1
```

Comment:
Was it intentional to remove `<!>` and `>>=`?

Reply:
These operators exist in FsToolkit. I don't think we need them here